location/views.py

```diff
@@ -397,11 +397,12 @@ def apartment_upload(request):
             # parse out the latitude and longitude from the response
             lat_lng = g_utils.parse_lat_lng(g_data[0])
 
-            g_locality = g_utils.get_city(g_data[0])[1]
+            g_city, g_locality = g_utils.get_city(g_data[0])
+            street_num, street = g_utils.get_address(g_data[0])
 
```

Comment:
![image](https://user-images.githubusercontent.com/18141004/70406598-4b658f00-1a0f-11ea-90a0-33c5ac8ca6ba.png)
Not sure if it is within this PR's scope.
But if you add the `state` and `city` in the address filed, it created a location with duplicate state and city.

Reply:
fixed

Reply:
Nice